roles/openshift_logging_elasticsearch/templates/es.j2

```diff
@@ -66,60 +94,78 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             -
-              name: "KUBERNETES_TRUST_CERT"
-              value: "true"
+              name: "NODE_NAME"
```

Comment:
I think this is a bad idea until we can fix how the searchguard index is populated.  This essentially reverts us to before where each ES node has its own index

Reply:
Is there something we need to do now?  I'd like to get this PR merged ASAP.